bin/pre-commit

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-PATH="$PATH:/usr/local/bin"
```

Comment:
Sure @raoulwegat , can you test https://github.com/Automattic/wp-calypso/pull/12548 works and green-lit it?